apps/ledger-live-mobile/src/families/ethereum/EditTransaction/Speedup.tsx

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
This page is a middleman as well just to get the transaction from `useBridgeTransaction`